x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -102,6 +108,12 @@ export interface PackageClient {
     prerelease?: false;
   }): Promise<PackageList>;
 
+  getAgentPolicyInputs(
+    pkgName: string,
+    pkgVersion?: string,
+    prerelease?: false
+  ): Promise<Array<Partial<FullAgentPolicyInput>>>;
```

Comment:
This new method in the package service is the equivalent of the existing Inputs REST API: 
```text
GET /api/fleet/epm/templates/{name}/{version}/inputs?format=json
```